birdhouse/components/stac/config/proxy/conf.extra-service.d/stac.conf.template

```diff
@@ -1,10 +1,10 @@
 
-    location /stac {
+    location /stac/ {
```

Comment:
I'm worried about this edit. 

I do not remember why it was omitting the `/`, but I remember having a lot of problems dealing with its redirects in STAC, because its own internal prefix redirect does weird stuff we cannot override explicitly, hence why there is dual `/stac` stuff happening. 

I would rather keep this a separate PR and test it more thoroughly.

Reply:
This has always bothered me and I'm 99% sure that this is an oversight and it always should have been `/stac/`.

But you're right, this isn't strictly related to this PR so I can remove it for now.